.github/workflows/publish.yml

```diff
@@ -24,15 +26,17 @@ jobs:
   test-publish:
     needs: [dist]
     name: Publish to TestPyPI
-    if: github.repository == 'PHOTOX/promdens'
     if: >-
       github.repository == 'PHOTOX/promdens' &&
-      github.ref_type != 'tag'
+      github.event_name == 'pull_request' &&
```

Comment:
These conditions are probably redundant, but it's better to be on the safe side I guess.